webdriver-spec.html

```diff
@@ -8071,8 +8071,6 @@
 <section>
 <h2>User Prompts</h2>
 
-<section><!-- Introduction -->
-  <h2>Introduction</h2>
 <p><i>This section is non-normative</i>
```

Comment:
I think this paragraph can just be dropped, actually. We don’t have it for all the other sections.